Doc/library/faulthandler.rst

```diff
@@ -66,10 +66,24 @@ Dumping the traceback
       Added support for passing file descriptor to this function.
 
 
+Dumping the C stack
+-------------------
+
+.. versionadded:: next
+
+.. function:: dump_c_stack(file=sys.stderr)
```

Comment:
Wouldn't it be better to have `file=None` and redirect it to sys.stderr if it's None? That way, we can patch `sys.stderr` locally (otherwise the default would be set once the module is loaded)

Reply:
That's what happens at the C level, yeah. `faulthandler` just denotes it like this because it's simpler for documentation.

Reply:
Mmmh. `print()` uses `None` for `sys.stdout` so I don't know whatis the preferred way. Up to you!

Reply:
Let's just keep it as-is. `faulthandler.enable` already uses `sys.stderr`.